conf/modules.config

```diff
@@ -275,7 +275,10 @@ process {
         ext.args2        = { params.use_gatk_spark && params.use_gatk_spark.contains('markduplicates') && (!params.skip_tools || (params.skip_tools && !params.skip_tools.contains('markduplicates'))) ? '-n' : '' }
         ext.prefix       = { params.split_fastq > 1 ? "${meta.id}".concat('.').concat(reads.get(0).name.findAll(/part_([0-9]+)?/).last()) : "" }
         publishDir       = [
-            enabled: false
+            enabled: (params.skip_tools && params.skip_tools.contains('markduplicates')),
```

Comment:
need to try more stuff, and probably add some conditions with `meta.size` not to save files that will be merged